sky/skylet/job_lib.py

```diff
@@ -330,6 +330,14 @@ def get_job_submission_port():
     return port
 
 
+def get_ray_port():
+    port_path = os.path.expanduser(constants.SKY_REMOTE_RAY_PORT_FILE)
```

Comment:
Nit: Can we document in constants.py what SKY_REMOTE_RAY_PORT_FILE means exactly? Does it always contain the currently active SkyPilot-internal Ray cluster's port (for old clusters, 6379; newer ones, 6380)?

Reply:
Added the comment.

Yes, we dump the info right after the `ray start` is called in the ray yaml file. The file will not exist if our ray cluster is on old 6379, because those clusters should be launched before #1790. That is why we have the file existence check and fallback to the default port. Added a docstr for these two functions as well.